loops/prime-numbers.cpp

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+#include <cmath>
+
+using namespace std;
+
+int main() {
+    int number;
+    bool isPrime = true, not01 = false;
+    cin >> number;
+    isPrime = (number == 0 || number == 1 ? isPrime = false : not01 = true);
```

Comment:
Тук може директно да се излезе от програмата.

```suggestion
    if(number < 2){
         cout<<"no"<<endl;
         return 0;
    }
```